clang/runtime/dpct-rt/include/dpct/math.hpp

```diff
@@ -2055,6 +2055,79 @@ class joint_matrix {
   matrix_accessor x;
   const size_t num_elements;
 };
+
+/// Multiplies 2 16x16 & 16x8 matrices and accumulates the result to a 16x8 b32
+/// matrix
```

Comment:
For this helper function to work correctly, the size of the subgroup for the kernel function must be 32, the description for this helper function should clarify this.

Reply:
Sure, I'll add the info in the comment

Reply:
I have the feeling the comment does not describe the case when C and D are different.